frontend/packages/console-app/src/actions/creators/hpa-factory.ts

```diff
@@ -93,22 +88,26 @@ type DeploymentActionExtraResources = {
 };
 
 export const useHPAActions = (kindObj: K8sKind, resource: K8sResourceKind) => {
-  const [namespace] = useActiveNamespace();
+  const namespace = resource?.metadata?.namespace;
+
   const watchedResources = React.useMemo(
-    () => ({
-      hpas: {
-        isList: true,
-        kind: HorizontalPodAutoscalerModel.kind,
-        namespace,
-        optional: true,
-      },
-      csvs: {
-        isList: true,
-        kind: referenceForModel(ClusterServiceVersionModel),
-        namespace,
-        optional: true,
-      },
-    }),
+    () =>
+      namespace
+        ? {
+            hpas: {
+              isList: true,
+              kind: HorizontalPodAutoscalerModel.kind,
+              namespace,
+              optional: true,
+            },
+            csvs: {
+              isList: true,
+              kind: referenceForModel(ClusterServiceVersionModel),
+              namespace,
+              optional: true,
+            },
+          }
+        : null,
     [namespace],
   );
   const extraResources = useK8sWatchResources<DeploymentActionExtraResources>(watchedResources);
```

Comment:
`extraResources` will be an empty object when no resources are passed to `useK8sWatchResources`. We need to review all references to this var and make sure this case is handled.